src/redux/store.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import spacereducer from './reducers';
```

Comment:
Please use camelcase in your codebase. camelcase make your code not clean.

Reply:
Fixed ✔